x-pack/plugins/profiling/public/views/add_data_view/index.tsx

```diff
@@ -263,197 +285,230 @@ enabled=1
+                iconType="gear"
+                fill
+                href={`${core.http.basePath.prepend(
+                  `/app/integrations/detail/profiler_agent-${data?.profilerAgent.version}/overview`
+                )}`}
               >
-                {i18n.translate('xpack.profiling.tabs.symbols.step2.instructions', {
-                  defaultMessage: 'Instructions here',
+                {i18n.translate('xpack.profiling.tabs.elasticAgentIntegration.step2.button', {
+                  defaultMessage: 'Manage Universal Profiling agent in Fleet',
                 })}
-              </EuiLink>
-            </EuiText>
-          ),
-        },
-        {
-          title: i18n.translate('xpack.profiling.tabs.symbols.step3', {
-            defaultMessage: 'Upload symbols',
-          }),
-          content: (
-            <div>
-              <EuiCodeBlock paddingSize="s" isCopyable>
-                {`./symbtool push-symbols executable -u "${symbolUrl}" -t <ES token> -e <my executable>`}
-              </EuiCodeBlock>
-              <EuiSpacer size="m" />
-              <EuiText>
-                <FormattedMessage
-                  id="xpack.profiling.tabs.symbols.step3.replace"
-                  defaultMessage="Replace {es_token} etc. with the actual values. You can pass {help} to obtain a list of other arguments."
-                  values={{
-                    es_token: <EuiCode>{`<ES token>`}</EuiCode>,
-                    help: <EuiCode>--help</EuiCode>,
-                  }}
-                />
-              </EuiText>
-              <EuiSpacer size="s" />
-              <EuiText>
-                <FormattedMessage
-                  id="xpack.profiling.tabs.symbols.step3.doc-ref"
-                  defaultMessage="Documentation for more advanced uses cases is available in {link}."
-                  values={{
-                    link: (
-                      <EuiLink
-                        data-test-subj="profilingAddDataViewTheCorrespondingDocumentationPageLink"
-                        target="_blank"
-                        href={`${core.docLinks.ELASTIC_WEBSITE_URL}/guide/en/observability/${core.docLinks.DOC_LINK_VERSION}/profiling-add-symbols.html`}
-                      >
-                        {i18n.translate('xpack.profiling.tabs.symbols.step3.doc-ref.link', {
-                          defaultMessage: 'the corresponding documentation page',
-                        })}
-                      </EuiLink>
-                    ),
-                  }}
-                />
-              </EuiText>
-            </div>
-          ),
-        },
-      ],
-    },
-  ];
+              </EuiButton>
+            ),
+          },
+        ],
+      },
+      {
+        key: AddDataTabs.Symbols,
+        title: i18n.translate('xpack.profiling.tabs.symbols.title', {
+          defaultMessage: 'Upload Symbols',
+        }),
+        subTabs: ['x86_64', 'arm64'].map((arch: string) => {
```

Comment:
To avoid duplication, let's extract this array to a const and reuse it above too.
You can place it out side the component.
```
const architectures = ['x86_64', 'arm64'];
```
```suggestion
        subTabs: architectures.map((arch) => {
```